lib/css/components/_stacks-widget-static.less

```diff
@@ -12,7 +12,7 @@
 //  _stacks-widget-dynamic.less.
 
 
-@s-sidebarwidget-content-padding: @su16 - 1px; // subtract 1px for border
+@s-sidebarwidget-content-padding: @su16 - 1; // subtract 1px for border
```

Comment:
No units?  What is this?

Reply:
Yeah, I don't know. It's at least consistent without the units. We do stuff like `@su2 / 2` all over Stacks.